api/index.js

```diff
@@ -3,12 +3,41 @@ const { REDIS_AUTH, REDIS_MAX_CLIENT, REDIS_READ_HOST, REDIS_READ_PORT, REDIS_WR
 const { SERVER_PROTOCOL, SERVER_HOST } = require('./config')
 const RedisConnectionPool = require('redis-connection-pool')
 const bodyParser = require('body-parser')
+const crypto = require('crypto')
 const express = require('express')
 const isProd = process.env.NODE_ENV === 'production'
+const jwt = require('jsonwebtoken')
+const jwtExpress = require('express-jwt')
+
 const router = express.Router()
 const superagent = require('superagent')
 
 const apiHost = API_PROTOCOL + '://' + API_HOST + ':' + API_PORT
+const consoleLogOnDev = ({ msg, showSplitLine }) => {
+  if (!isProd) {
+    showSplitLine && console.log('####################')
+    console.log(msg)
+    showSplitLine && console.log('####################')
+  }
+}
```

Comment:
console log in production?

Reply:
no, it console logs only when it is on dev or local server